dashboard/lib/pd/workshop_survey_constants.rb

```diff
@@ -32,6 +33,17 @@ module WorkshopSurveyConstants
       SUBJECT_CSD_UNITS_1_3 => ACADEMIC_YEAR_1_2_CATEGORY,
       SUBJECT_CSD_UNITS_4_6 => ACADEMIC_YEAR_3_4_CATEGORY,
       SUBJECT_CSD_TEACHER_CON => LOCAL_CATEGORY,
+      SUBJECT_CSF_201 => CSF_CATEGORY,
+    }
+
+    CSF_SURVEY_NAMES = [
+      PRE_DEEPDIVE_SURVEY = 'pre201',
+      POST_DEEPDIVE_SURVEY = 'post201',
+    ]
+
+    CSF_SURVEY_INDEXES = {
+      PRE_DEEPDIVE_SURVEY => 0,
+      POST_DEEPDIVE_SURVEY => 1,
     }
```

Comment:
```rb
    CSF_SURVEY_INDEXES = CSF_SURVEY_NAMES.each_with_index.to_h
```

Reply:
I make this explicit on purpose. The survey index will be saved to `day` column in `WorkshopDailySurvey` table. The mapping between indexes to survey names (or types) should be explicitly defined and not change if `CSF_SURVEY_NAMES` entries order changes in the future . Will add a comment explaining this.

Reply:
Got it! I was wondering, it just looks derived in the example case but is not actually a derived index.